pkg/operator/ceph/cluster/osd/osd_pvc_test.go

```diff
@@ -112,7 +107,7 @@ func testOSDsOnPVC(t *testing.T) {
 		// can't use mockNodeOrchestrationCompleted here b/c it uses a context.CoreV1() method that
 		// is mutex locked when a reactor is processing
 		status := parseOrchestrationStatus(cm.Data)
-		infof(t, "configmap reactor: updating configmap %q status to completed", cm.Name)
+		t.Logf("configmap reactor: updating configmap %q status to completed", cm.Name)
```

Comment:
Does `t.Log()` offer something over `logger.Infof()` like we use elsewhere?

Reply:
It differentiates logs that come from the test framework from logs that come from Rook code.

Reply:
Sounds good, probably a good habit to use in other tests going forward as well